client/store/products.js

```diff
@@ -1,14 +1,21 @@
-export const GET_PRODUCTS = 'GET_PRODUCTS'
-export const ADD_PRODUCT = 'ADD_PRODUCT'
 import axios from 'axios'
 
 const initialState = []
```

Comment:
```suggestion
const initialState = {allProducts:[],selectedProduct:{}}
```

As written, when we fetch a single product, we will overwrite all of the products that we initially fetched from the DB. Instead, we can change what state looks like within the products reducer.